lib/src/parser.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:test_cov_console/src/parser_constants.dart';
+
+class Parser {
+  final List<String> arguments;
+
+  Parser(this.arguments);
+
+  Map<String, dynamic> parse() {
+    if (arguments.isEmpty) {
+      return {};
+    }
+
+    final Map<String, dynamic> result = {};
+    bool bContinue = false;
+    for (var i = 0; i < arguments.length; ++i) {
```

Comment:
reduce/fold is preferable since we're expecting returns from this iteration

Reply:
Updated please checked.